src/cdk/tree/tree.ts

```diff
@@ -154,15 +159,32 @@ export class CdkTree<T, K = T> implements AfterContentChecked, CollectionViewer,
     end: Number.MAX_VALUE,
   });
 
+  // TODO(cassc): will be used in a future PR
```

Comment:
It's a little tricky to understand the use of this without seeing it in the current PR - in future PRs it'd be easier if we delay adding variables until they are used

Reply:
Makes sense, will do.